rott/rt_cfg.c

```diff
@@ -428,6 +428,14 @@ boolean ParseConfigFile (void)
       ReadInt("ScreenWidth", &iGLOBAL_SCREENWIDTH);
       ReadInt("ScreenHeight", &iGLOBAL_SCREENHEIGHT);
 
+	if ((iGLOBAL_SCREENWIDTH != 320 && iGLOBAL_SCREENWIDTH != 640) ||
```

Comment:
This would still let 640x200 pass through?

Reply:
Oh, derp